From: Sophie Brun <sophie@offensive-security.com>
Date: Thu, 18 Jun 2020 11:04:58 +0200
Subject: Change path for minizip/ioapi.h

Last-Update: 2020-07-30
Description: We didn't include contrib/minizip/ioapi.h (upstream ships
it in deps/zlib). We can use the Debian package libminizip-dev: change
the include accordingly.
---
 include/ext_lzma.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/ext_lzma.h b/include/ext_lzma.h
index 194067b..ab0ca60 100644
--- a/include/ext_lzma.h
+++ b/include/ext_lzma.h
@@ -9,8 +9,8 @@
 #include <LzmaDec.h>
 #include <Lzma2Dec.h>
 
-#include "minizip/ioapi.h"
-#include "minizip/unzip.h"
+#include <minizip/ioapi.h>
+#include <minizip/unzip.h>
 
 int hc_lzma1_decompress (const unsigned char *in, SizeT *in_len, unsigned char *out, SizeT *out_len, const char *props);
 int hc_lzma2_decompress (const unsigned char *in, SizeT *in_len, unsigned char *out, SizeT *out_len, const char *props);
